src/hex_pb_package.erl

```diff
@@ -1,6 +1,6 @@
 %% -*- coding: utf-8 -*-
 %% Automatically generated, do not edit
-%% Generated by gpb_compile version 4.3.1
```

Comment:
It seems I generated this using an older version. This dependency is from a plugin (and not in the lock?) so I am not sure how to update it.